File: Modul-1/Python/PRAK110-2410817210001-MuhammadFadillah.py
import math

alas = 5
tinggi = 12

sisi_c = math.sqrt(alas**2 + tinggi**2)

keliling = alas + tinggi + sisi_c

luas = (alas * tinggi) / 2

print("Diketahui :")
print(f"Alas = {alas} cm")
print(f"Tinggi = {tinggi} cm")

print("\nJawab:")
print(f"Sisi A = {alas} cm")
print(f"Sisi B = {tinggi} cm")
print(f"Sisi C = {int(sisi_c)} cm")
print(f"Keliling = {int(keliling)} cm")
print(f"Luas = {int(luas)} cm")

File: Modul-1/C/PRAK110-2410817210001-MuhammadFadillah.c
#include <stdio.h>
#include <math.h>

int main() {
  int alas = 5;
  int tinggi = 12;

  int sisi_c = sqrt(alas * alas + tinggi * tinggi);

  int keliling = alas + tinggi + sisi_c;

  int luas = (alas * tinggi) / 2;

  printf("Diketahui :\n");
  printf("Alas = %d cm\n", alas);
  printf("Tinggi = %d cm\n", tinggi);

  printf("\nJawab:\n");
  printf("Sisi A = %d cm\n", alas);
  printf("Sisi B = %d cm\n", tinggi);
  printf("Sisi C = %d cm\n", sisi_c);
  printf("Keliling = %d cm\n", keliling);
  printf("Luas = %d cm\n", luas);

  return 0;
}